### ISSUE TITLE
PBC box information not loaded in `load_parmed` function

### ISSUE THREAD

**Issue by wwilla7** [Amber-MD/pytraj#1514]
I installed pytraj and parmed using the following commands:
```
conda create -n test -c ambermd pytraj ambertools
conda activate test
```

and then I ran in a python prompt

```
import pytraj as pt
import parmed as md
structure_pmd = pmd.load_file("bcd_pbc.prmtop", "bcd_pbc.rst7", structure = True)
print(structure_pmd.box)
traj_pmd = pt.load_parmed(structure_pmd)
print(traj_pmd.unitcells)
print(pt.distance(traj_pmd, ":1 :148", image=True))
```
> [44.537351 44.146274 44.499725 90.       90.       90.      ]
> None
> array([nan])

and I got `None` for unitcells, and a `nan` as my distance.

However, if I try this:
```
import pytraj as pt
traj_pt = pt.iterload("bcd_pbc.rst7", top = "bcd_pbc.prmtop")
print(traj_pmd.unitcells)
print(pt.distance(traj_pmd, ":1 :148", image=True))

```
> [[44.537351 44.146274 44.499725 90.       90.       90.      ]]
> array([28.83258138])

Using the same structure files, I got unitcells information here and a real distance between these two atoms.

I am not very familiar with trajectory file format, so I am wondering if these are some history behind it to not load a box. I noticed that the box was loaded from the rst7 file, but it was only stored in the `traj.top` object. 

For others who encounter this issue, I was able to perform distance calculations using the following workaround. I could open a PR to add something like this to the `load_parmed` function if that would be helpful:
```
traj_pmd.unitcells= np.hstack([[[traj_pmd.top.box[0],traj_pmd.top.box[1],traj_pmd.top.box[2],
                         90,90,90]]]).astype('f8')
```
[files.zip](https://github.com/Amber-MD/pytraj/files/4131930/files.zip)

**Comment by hainm** [comment]
thanks @wwilla7. We will investigate this issue.

**Comment by wwilla7** [comment]
Thanks, @hainm. I can use this workaround for my workflow, so this is not urgent to fix. But other users may find it useful.

**Comment by hainm** [comment]
hi @wwilla7, this issued is fixed in #1537. Thanks again for your report.

**Comment by wwilla7** [comment]
Hi @hainm, thanks!
